src/pages/waiting-room-to-car/cat-a-mod2/components/vehicle-checks-modal/__tests__/vehicle-checks-modal.cat-a-mod2.analytics.effects.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { TestBed } from '@angular/core/testing';
-import { VehicleChecksModalCatAMod2AnalyticsEffects } from '../vehicle-checks-modal.cat-a-mod2.analytics.effects';
+import { SafetyAndBalanceModalCatAMod2AnalyticsEffects } from '../vehicle-checks-modal.cat-a-mod2.analytics.effects';
```

Comment:
needs a rename so does the modal and all other associated files

Reply:
I think we should be keeping it Vehicle checks when I think about it. They are still Vehicle checks, it's just Safety and Balance not Show me and Tell me questions.

Reply:
Exporting `SafetyAndBalanceModal*` from `vehicle-checks-modal.*` is not something that we should do. The file or the class needs a rename.

Reply:
I ok with not renaming I'm with szabi the class and file name should be the same.

I still feel the reducer/store side needs a rename as it's not looking at the vehicle checks key in the state (that doesn't exist) it looks at the safety questions key and I think that's another thing we should keep in sync file name/json key

Reply:
My view is that we should stick with the vehicle chacks naming. I agree about the state, but that's a schema thing and for me should have stayed vehicle checks there too!